### DISCUSSION TITLE
How to hide Client IP address

### DISCUSSION THREAD

**Discussion by drtiwarir** [DNSCrypt/dnscrypt-proxy#2238] [Q&A]
Hi
I have tried DNSCrypt to watch streaming websites from India.
I used 2 different servers and setup Anonymized DNS relays, but my IP address is revealed and the streams don't work.
Can you provide me a tutorial/guide how to do it ?

Thanks

**Comment by jedisct1** [comment]
You probably need a VPN for this.

`dnscrypt-proxy` can hide the IP address for DNS queries, not when connecting to websites.
